.github/workflows/_test_requiring_torch_cuda.yml

```diff
@@ -0,0 +1,129 @@
+name: xla-test-requiring-torch-cuda
```

Comment:
minor suggestion: maybe name this one after the test objective? e.g. `test_cuda_interop` or something like that

Reply:
Thanks for the suggestion. It turned out more features than interoperability need torch CUDA, for example triton, etc.